tests/conftest.py

```diff
@@ -3662,7 +3660,6 @@ def bobby_columnar_table_multi_batch(empty_data_context):
         ExpectationConfiguration(
             **{
                 "expectation_type": "expect_column_values_to_match_regex",
-                "meta": {"notes": {"format": "markdown", "content": None}},
```

Comment:
Why was this necessary? Why didn't our tests fail before you did this?

Reply:
@NathanFarmer I saw a `PyCharm` error -- duplicated key; so I deleted it.  Our tests did not fail, because the dictionary does not forbid specifying the same key multiple times -- the last one is used.  This is one of those typos that an IDE catches for us.  Thanks.